components/catalogs/back/src/Infrastructure/Normalizer/CatalogNormalizer.php

```diff
@@ -35,7 +35,7 @@ public function normalize(mixed $object, ?string $format = null, array $context
         return [
             'id' => $object->getId(),
             'name' => $object->getName(),
-            'enabled' => false,
+            'state' => $object->getState(),
```

Comment:
```suggestion
            'enabled' => $object->isEnabled(),
```

I don't think we should leak the state